projects/plugins/jetpack/extensions/blocks/map/map.php

```diff
@@ -165,3 +165,48 @@ function render_single_block_page() {
 	exit;
 }
 add_action( 'wp', __NAMESPACE__ . '\render_single_block_page' );
+
+/**
+ * Helper function to generate the markup of the block in PHP.
+ *
+ * @param Array $points - Array containing geo location points.
+ *
+ * @return string Markup for the jetpack/map block.
+ */
+function map_block_from_geo_points( $points ) {
+	$map_block_data = array(
+		'points'    => $points,
+		'zoom'      => 8,
+		'mapCenter' => array(
+			'lng' => $points[0]['coordinates']['longitude'],
+			'lat' => $points[0]['coordinates']['latitude'],
+		),
+	);
+
+	$list_items = array_map(
+		function ( $point ) {
+			$link = add_query_arg(
+				array(
+					'api'   => 1,
+					'query' => $point['coordinates']['latitude'] . ',' . $point['coordinates']['longitude'],
+				),
+				'https://www.google.com/maps/search/'
+			);
+			return sprintf( '<li><a href="%s">%s</a></li>', $link, $point['title'] );
```

Comment:
Do you think it would be worth escaping the final URL, just in case we get odd coordinate values?

```suggestion
			return sprintf( '<li><a href="%s">%s</a></li>', esc_url( $link ), $point['title'] );
```